pkg/proc/gdbserial/rr.go

```diff
@@ -178,7 +178,7 @@ func (err ErrPerfEventParanoid) Error() string {
 	return fmt.Sprintf("rr needs /proc/sys/kernel/perf_event_paranoid <= 1, but it is %d", err.actual)
 }
 
-func checkRRAvailabe() error {
+func CheckRRAvailable() error {
```

Comment:
This change can be reverted now.

Reply:
Ok, may it be a separate pr? it a typo after all and it wouldn't hurt to have it fixed

Reply:
(the typo, the method may remain unexported)

Reply:
Oh, I didn't even notice the typo. I guess just unexport the function then.